src/Artemis.UI/Screens/Debugger/Tabs/Logs/LogsDebugView.axaml

```diff
@@ -2,15 +2,24 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:aedit="https://github.com/avaloniaui/avaloniaedit"
+             xmlns:controls="clr-namespace:Artemis.UI.Controls"
+             xmlns:logs="clr-namespace:Artemis.UI.Screens.Debugger.Logs"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
-             x:Class="Artemis.UI.Screens.Debugger.Logs.LogsDebugView">
-    <StackPanel>
-        <TextBlock Classes="h3">Logs</TextBlock>
-        <TextBlock TextWrapping="Wrap">
-            On this page you can view Artemis's logs in real-time. Logging can come from Artemis itself, plugins and scripts.
-        </TextBlock>
+             x:Class="Artemis.UI.Screens.Debugger.Logs.LogsDebugView"
+             x:DataType="logs:LogsDebugViewModel">
+  <aedit:TextEditor Name="log"
+                    Document="{ CompiledBinding Document }"
+                    IsReadOnly="True"
+                    FontFamily="Consolas"
+                    FontSize="12"
+                    HorizontalScrollBarVisibility="Auto"
```

Comment:
Lets add `Padding="0 15 15 15"` here so that the scroll bars don't overlap with the text